src/types/algorand-client.spec.ts

```diff
@@ -111,20 +112,19 @@ describe('AlgorandClient', () => {
   })
 
   test('method with method call arg', async () => {
-    const helloWorldParams = {
-      type: 'methodCall' as const,
+    const helloWorldCall = {
       sender: alice.addr,
       appId: appId,
       method: appClient.appClient.getABIMethod('helloWorld')!,
-    }
+    } satisfies MethodCallParams
```

Comment:
You don't /have/ to use `satisfies MethodCallParams`, but by doing so you'd get more obvious intellisense.